packaging/nnstreamer.spec

```diff
@@ -102,7 +104,23 @@ pushd build
 popd
 
 pushd tests
-./testAll.sh
+# armv7l needs 6~7min to run 'testAll.sh', so timeout value is set 10min
```

Comment:
before: 
`# armv7l needs 6~7min to run 'testAll.sh', so timeout value is set 10min`

after: 
```
# The 'testAll.sh' script requires 6~7min to run armv7l  binary files in the current CI server. 
# The timeout value is  10min as a heuristic value from oure experience.
```

Reply:
fixed.